lib/embed/embed_this_panel.rb

```diff
@@ -0,0 +1,58 @@
+module Embed
+  class EmbedThisPanel
+    def initialize(druid:, height:, width:, purl_object_title:, &block)
+      @druid = druid
+      @height = height
+      @width = width
+      @purl_object_title = purl_object_title
+      @panel_content = yield block
+    end
+
+    def to_html
+      <<-HTML.strip_heredoc
+        <div class='sul-embed-panel-container'>
+          <div class='sul-embed-panel sul-embed-embed-this-panel' style='display:none' aria-hidden='true'>
+            <div class='sul-embed-panel-header'>
+              <button class='sul-embed-close' data-sul-embed-toggle='sul-embed-embed-this-panel'>
+                <span aria-hidden='true'>&times;</span>
+                <span class='sul-embed-sr-only'>Close</span>
+              </button>
+              <div class='sul-embed-panel-title'>Embed</div>
+            </div>
+            <div class='sul-embed-panel-body'>
+              <div class='sul-embed-embed-this-form'>
+                <span class='sul-embed-options-label'>Select options:</span>
+                <div class='sul-embed-section sul-embed-embed-title-section'>
+                  <input type='checkbox' id='sul-embed-embed-title' 'data-embed-attr'='hide_title' checked=true />
```

Comment:
The `data-embed-attr` should not have the single quotes around it.  This is currently causing the title option to add the parameter `undefined=true` instead of `hide_title=true` (there should have been a test for this, looks like that one got away from us).

<img width="542" alt="undefined-title" src="https://cloud.githubusercontent.com/assets/96776/14578189/42ae2d34-0336-11e6-833e-fd226dccb511.png">